java/src/test/java/name/fraser/neil/plaintext/diff_match_patch_test.java

```diff
@@ -18,9 +18,11 @@
 
 /**
  * Compile from diff-match-patch/java with:
- * javac -d classes src/name/fraser/neil/plaintext/diff_match_patch.java tests/name/fraser/neil/plaintext/diff_match_patch_test.java
+ * javac -d target/classes src/main/java/name/fraser/neil/plaintext/diff_match_patch.java
+ * javac -classpath target/classes -d target/test-classes src/test/java/name/fraser/neil/plaintext/diff_match_patch_test.java
```

Comment:
This second javac line gives me:
javac: directory not found: target/test-classes

Also, corresponding changes are needed in Speedtest.java.

More generally, is there a reason why this directory tree is getting deeper and more complex?  Is Maven unable to handle the previous structure?

Reply:
It is possible to configure maven to adopt previous structure. But how it is configured it is the standard. Also, since it is maven enabled, we would use maven to compile test. But since it is not Junit enabled, tests will not be picked to be run by Maven. I can convert the tests to Junit so you can just test with `mvn test`